party-role/src/main/resources/application.yaml

```diff
@@ -0,0 +1,48 @@
+micronaut:
+  application:
+    name: ${project.artifactId}
+
+  caches:
```

Comment:
There were some changes in application properties. Now there are multiple properties for different environments. Could you adjust it? (As example, you can check account-module)